server/chat-commands/core.ts

```diff
@@ -374,12 +377,17 @@ export const commands: Chat.ChatCommands = {
 
 		return this.parse(message);
 	},
-	msghelp: [`/msg OR /whisper OR /w [username], [message] - Send a private message.`],
+	msghelp: [
+		`/msg OR /whisper [username], [message] - Send a private message.`,
+		`You can also use /whisper, /w, /dm, or /pm. You can also click a username and press Chat.`,
+		`If the other user is offline, use /offlinemsg.`
+				],
```

Comment:
If we are adding this much text, might as well link to https://pokemonshowdown.com/privacy

Reply:
I don't think there's a need to list all these aliases.